pub/index.html

```diff
@@ -1,221 +1,154 @@
-<!doctype html>
-<html class="no-js" lang="">
+<!--
+1. 1/14/2014 23:55pm
+    Replaced filler icons
+    Inserted Lipsum
+2. 1/14/2014 12:24am
+    Starting css
+    Changed fonts
+    Started positioning icons
+    Rounded post corners
+    Reversed icon colors
+    Changed time and date colors
+    Move time up to make room for       caret (someday)
+
+
+
+-->
```

Comment:
Interesting. Were these your Pomodoro notes? They'd make great commit messages... Github will even render Markdown in your extended commit messages as HTML, so you can use checklists, etc. Try it!